### PR TITLE
fullwidth for buttons. dont shrink waiting buttons

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
- [ ] merge after release

File: shared/common-adapters/button.js [hunk 1]
```diff
@@ -70,30 +73,27 @@ class Button extends React.Component<Props> {
...
+    if (this.props.waiting) {
       labelStyle = collapseStyles([labelStyle, styles.opacity0])
     }
 
     containerStyle = collapseStyles([containerStyle, this.props.style])
 
     const onClick = (!this.props.disabled && !this.props.waiting && this.props.onClick) || null
-
-    const whiteSpinner = this.props.type !== 'PrimaryGreenActive'
+    const whiteSpinner = !(this.props.type === 'PrimaryGreenActive' || this.props.type === 'Secondary')
```
Comment (buoyad) [inline shared/common-adapters/button.js]: :sheep:   `!['PrimaryGreenActive', 'Secondary'].includes(this.props.type)`

Comment (chrisnojima) [reply shared/common-adapters/button.js]: i've found this form to confuse flow so i've avoided it.unsure if newer flow is smarter about this

Comment (buoyad) [reply shared/common-adapters/button.js]: ah np, that's too bad, it's very convenient for whitelists

Comment (chrisnojima) [reply shared/common-adapters/button.js]: https://flow.org/try/#0C4TwDgpgBAglC8UDkB7AdhJUA+zgHcUkAoYgY3QGdgoA3BZACwgBsWV8TiAzAVzTLAAluijcAFAEMAXLACUUAN7EoUCmkooWEAHTsA5lLnEAvqSHco4+vFvJ0mHLht2kBIguWqJtY2eIWVgDaqBhIADR4hEgAujpCAiy8ACYQlNZynipiGaZAA

**Pull request merged**